src/main/java/com/vinylteam/vinyl/entity/ConfirmationToken.java

```diff
@@ -0,0 +1,69 @@
+package com.vinylteam.vinyl.entity;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+import java.util.UUID;
+
+public class ConfirmationToken {
+
+    private long id;
+    private long userId;
+    private UUID token;
+    private Timestamp timestamp;
+
```

Comment:
Database uses Timestamp - when you use LocalDateTime, you still work with Timestamp from result set and have to convert it in LocalDateTime.
I decided not to bother with it, because there is a literal java.sql.Timestamp class for this.